src/main/java/org/terasology/marcinsc/blockFamily/FullRotationFamily.java

```diff
@@ -20,47 +20,48 @@
 import org.terasology.math.Side;
 import org.terasology.math.geom.Vector3i;
 import org.terasology.naming.Name;
-import org.terasology.world.BlockEntityRegistry;
-import org.terasology.world.WorldProvider;
 import org.terasology.world.block.Block;
+import org.terasology.world.block.BlockBuilderHelper;
 import org.terasology.world.block.BlockUri;
 import org.terasology.world.block.family.AbstractBlockFamily;
+import org.terasology.world.block.family.RegisterBlockFamily;
+import org.terasology.world.block.loader.BlockFamilyDefinition;
 
 import java.util.Map;
 
+@RegisterBlockFamily("fullRotation")
 public class FullRotationFamily extends AbstractBlockFamily implements RotationBlockFamily {
-    private Map<Rotation, Block> blocks;
-
+    private Map<Rotation, Block> blocks = Maps.newHashMap();
+    private Map<BlockUri, Block> blockUriMap = Maps.newHashMap();
     private Block archetypeBlock;
 
-    private Map<BlockUri, Block> blockUriMap = Maps.newHashMap();
+    public FullRotationFamily(BlockFamilyDefinition definition, BlockBuilderHelper helper) {
+        super(definition, helper);
+        super.setCategory(definition.getCategories());
```

Comment:
Why are the calls to `setCategory` and `setBlockUri` on `super`? In this case it does not make a huge difference, but one should not bypass possibly different implementations in the class, so I would rather use `this.setCategory(...)`.